packages/ws-server/tests/acceptance/call.spec.ts

```diff
@@ -65,8 +67,21 @@ describe('@release @web-socket eth_call', async function () {
     erc20EtherInterface: ethers.Interface;
 
   before(async () => {
-    accounts[0] = await global.servicesNode.createAliasAccount(100, global.relay.provider, requestId);
-    await new Promise((r) => setTimeout(r, 1000)); // wait for accounts[0] to propagate
+    requestId = Utils.generateRequestId();
```

Comment:
this block seems repetitive in ws-server acceptance tests, we should refactor and group them in a util function where makes sense.

Reply:
Agree, and we are not done with refactoring, let's do it in another PR, as this one got pretty big.